handlers/new-product-api/src/main/scala/com/gu/newproduct/api/addsubscription/validation/supporterplus/AmountLimits.scala

```diff
@@ -0,0 +1,58 @@
+package com.gu.newproduct.api.addsubscription.validation.supporterplus
+
+import com.gu.i18n.Currency
+import com.gu.i18n.Currency._
+import com.gu.newproduct.api.productcatalog.PlanId
+import com.gu.newproduct.api.productcatalog.PlanId.AnnualSupporterPlus
+
+case class SupporterPlusLimits(monthly: AmountLimits, annual: AmountLimits)
+
+case class AmountLimits(min: Int, max: Int)
+
+object AmountLimits {
+
```

Comment:
stripping off the `00` from each via a utility constructor would make it much easier to check they are reasonable
```suggestion
  def fromMajorUnits(min: Int, max: Int) = AmountLimits(min * 100, max * 100)
```